mcp/src/main/java/io/modelcontextprotocol/spec/McpServerTransportProvider.java

```diff
@@ -42,11 +42,11 @@ public interface McpServerTransportProvider {
 	/**
 	 * Sends a notification to all connected clients.
 	 * @param method the name of the notification method to be called on the clients
-	 * @param params a map of parameters to be sent with the notification
+	 * @param params parameters to be sent with the notification
 	 * @return a Mono that completes when the notification has been broadcast
 	 * @see McpSession#sendNotification(String, Map)
 	 */
-	Mono<Void> notifyClients(String method, Map<String, Object> params);
+	Mono<Void> notifyClients(String method, Object params);
```

Comment:
Let's keep in mind to take note of the breaking change in the release docs. I think it's worth the effort though.